quinn-proto/src/connection/mod.rs

```diff
@@ -2789,10 +2795,10 @@ where
         self.streams.alloc_remote_stream(&self.peer_params, dir);
     }
 
-    fn add_read_credits(&mut self, id: StreamId, more: bool) {
+    fn add_read_credits(&mut self, id: StreamId, max_stream_data_update: bool) {
```

Comment:
I'm not sure this is much clearer. Maybe an imperative form (e.g. `update_max_stream_data`) rather than passive? Or something more abstract, like `more_data_expected`?

Reply:
You are right. I updated to the first. What I don't like about `more_data_expected` is that this would also be true if we don't want transmit a `MAX_STREAM_DATA` frame.

We can also add the transmission part, like `transmit_max_stream_data`. or `require_max_stream_data_update`?

Reply:
I think I like `transmit` more than `update`, good call.